LICENSE.nextstrain-ncov-ingest

```diff
@@ -0,0 +1,28 @@
+This license applies to the original copy of "source-data/gisaid_geoLocationRules.tsv" 
+from the Nextstrain ncov-ingest project into this project as 
+"augur/data/geolocation_rules.tsv".
+
+Any subsequent modifications to this project's copy are licensed under the 
+license of this project, not of Nextstrain ncov-ingest.
+
+MIT License
+
+Copyright (c) 2020 Nextstrain
```

Comment:
should probably [strip the year](https://github.com/nextstrain/public/issues/2) here

Reply:
Ah, good to know. Also added ncov-ingest to the checklist in the issue.

Reply:
Ah, actually how would this work? 
The original copyright still includes the year, so should this copy still include the year until the original removes it?

Reply:
Yeah, license file should reflect the original. I'd leave as-is for this PR.